test/indexer/test_indexer.py

```diff
@@ -467,6 +469,28 @@ def test_zero_tallies(self):
                                     fr'.*_aggregate/_create/{doc_id}.*')
             self.assertTrue(any(message_re.fullmatch(message) for message in logs.output))
 
+    def test_entity_dijunction(self):
+        bundle1 = self._load_canned_bundle(self.old_bundle)
+        self._index_bundle(bundle1)
+        # bundle2 we create from bundle1's contents, however it gets a different
+        # source and bundle UUID, giving us entities from multiple sources.
+        source = DSSSourceRef.for_dss_source('https://fake_dss_instance2/v1:/2')
+        bundle_uuid = '3ccbc8a7-009e-4bc6-8854-da47bffae862'
```

Comment:
Why not inline? Like in 
 https://github.com/DataBiosphere/azul/pull/7410/commits/ca21c2610507202253f984c89e96d1f2c091a4e4#diff-ab292a23d1637ee493050d47b8a5881c6e9254367450ed77971ae8005466b7d4R543.